installation/docker-commands/create.sh

```diff
@@ -8,16 +8,6 @@ function pause() {
 echo
 echo "** ✏️  Creating a new Hummingbot instance **"
 echo
-# Specify hummingbot version
-echo "ℹ️  Press [enter] for default values."
-echo
-echo "➡️  Enter Hummingbot version: [latest|development] (default = \"latest\")"
-read TAG
-if [ "$TAG" == "" ]
-then
-  TAG="latest"
```

Comment:
Could leave TAG variable so user can select desired version in future.

Option 1. Set default TAG to an available version.
```
if [ "$TAG" == "" ]; then
  TAG=0.22.0
```

Option 2. Exit with message
```
if [ "$TAG" == "" ]; then
  echo "version not set"
  exit 1
```

Option 3. Leave the original code. Upload a docker image with tag `latest`.

Reply:
chosed option 1. thx

Reply:
I chose the option1. solved. thx